### PR TITLE
add most@1.0.0 w/ npm auto-update

### LINKED ISSUE #8143: [Request] Add most.js
**Library name:** most.js
**Git repository url:** https://github.com/cujojs/most
**npm package url(optional):** https://www.npmjs.com/package/most
**License(s):** MIT
**Official homepage:** https://github.com/cujojs/most
**Wanna say something? Leave message here:**
# 

Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #8143 
@dannyxx001 please help me review this pull request, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Profile of the lib
- Git repository (required): https://github.com/cujojs/most
- Official website (optional, not the repository):
- NPM package url (optional): https://www.npmjs.com/package/most
- GitHub / Bitbucket popularity (required):
  - Count of stars: 1,043
  - Count of watchers: 61
  - Count of forks: 58
- NPM download stats (optional):
  - Downloads in the last day: 17
  - Downloads in the last week: 863
  - Downloads in the last month: 3226
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [x] This lib was not found on cdnjs repo
- [x] No already exist / duplicated issue and PR
- [x] The lib has notable popularity
  - [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [x] More than 500 downloads stats per month on npm registry
- [ ] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.
# Git commit checklist
- [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [x] The parent of the commit(s) in the PR is not old than 3 days.
- [x] Pull request is sending from a non-master branch with meaningful name.
- [x] Separate unrelated changes into different commits.
- [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [x] Close corresponding issue in commit message
- [x] Mention related issue(s), people in commit message, comment.

close #8143, cc @briancavalier

### CODE REVIEW

**Comment by briancavalier** [comment]
Thanks for doing this, @kennynaoh!  Let me know if I can help in any way.

**Comment by dannyxx001** [comment]
I think this PR is OK. @PeterDaveHello please help us review again, thanks.

File: ajax/libs/most/package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  "license": "MIT",
+  "dependencies": {
+    "@most/multicast": "^1.1.1",
+    "@most/prelude": "^1.2.0",
+    "symbol-observable": "^0.2.4"
+  },
+  "npmName": "most",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
```
Comment (PeterDaveHello) [inline ajax/libs/most/package.json]: I didn't see `dist` folder in npm package `most` v0.19.7, @kennynaoh @dannyxx001 are you sure about this?

Comment (dannyxx001) [inline ajax/libs/most/package.json]: Sorry for my laziness. I don't really install npm package for reviewing. My fault Orz

**Comment by kennynaoh** [comment]
Sorry for wrong `npmFileMap`.
@dannyxx001 
I have fixed it.
Please help me review it again.
Thanks!

**Comment by kennynaoh** [comment]
ping @dannyxx001

File: ajax/libs/most/0.19.7/most.js [hunk 2]
```diff
@@ -0,0 +1,712 @@
+/** @license MIT License (c) copyright 2010-2016 original author or authors */
+/** @author Brian Cavalier */
+/** @author John Hann */
+
+var Stream = require('./lib/Stream');
+var base = require('@most/prelude');
+var core = require('./lib/source/core');
+var from = require('./lib/source/from').from;
+var periodic = require('./lib/source/periodic').periodic;
+var symbolObservable = require('symbol-observable');
```
Comment (dannyxx001) [inline ajax/libs/most/0.19.7/most.js]: Sorry for late response. I think it needs lib files.

**Comment by kennynaoh** [comment]
@x09326 
I have added some libraries it need.
Please help me review it again.
Thanks!

**Comment by x09326** [comment]
@kennynaoh 
Do you confirm the files with the author?
I'm not sure whether the files are needed or not.

**Comment by kennynaoh** [comment]
close due to existing on cdnjs https://cdnjs.com/libraries/most

cc @PeterDaveHello

**Comment by PeterDaveHello** [comment]
@kennynaoh you didn't close it ...

**Pull request closed without merging**